cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/actor/ProgressiveSolver.scala

```diff
@@ -58,11 +58,12 @@ class ProgressiveSolver(val dataManager: ActorRef,
       val boundary = new TInterval(min, max)
 
       val initialDuration = config.FirstQueryTimeGap
+      val minDuration = config.MinTimeGap
```

Comment:
minDuration is confusing. (duration in minutes OR minimum duration?) We know it's minimum duration. So, let's call it minimumDuration.

Reply:
Sure.